frontend/packages/console-shared/src/components/form-utils/FormFooter.tsx

```diff
@@ -18,9 +19,15 @@ const FormFooter: React.FC<FormFooterProps> = ({
   successMessage,
   disableSubmit,
   showAlert,
+  sticky,
+  submitTestId,
+  resetTestId,
+  cancelTestId,
```

Comment:
Instead of accepting multiple test IDs. A better pattern is to only accept a single `data-test-id` prop and assign the incoming test ID to the context root of this component (in this case the `ButtonBar`) and then assign static test IDs to the individual buttons: `submit-button`, `reset-button`. Remember that test IDs only need to be unique within the current context. A new context can be established anywhere and test ID selectors often traverse multiple contexts.

eg. `git-import -> form -> form-footer -> submit-button`

Furthermore protractor does have selectors `buttonText` which can be used as well.

Reply:
Sure, i will update the code and e2e tests (if any). This seems to be a better approach for test ids.